00-HTML-CSS-basics/The basics of CSS/2.5.2 Flexbox/style.css

```diff
@@ -0,0 +1,94 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    background-color: #eeeeee;
+    color: #000000;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.container{
+    /*border: solid 5px black;*/
+    padding: 10px;
+    display: flex; 
+    justify-content: space-around;
+    align-items: center;
+    flex-wrap: wrap; /* Establece los ítems en modo multilínea (permite que se desborde el contenedor). */
+    margin: 0px 25px;
+}
+
+/*
+
+    justify-content: Se utiliza para alinear los ítems del eje principal (por defecto, el horizontal).
+    align-items: Usada para alinear los ítems del eje secundario (por defecto, el vertical).
+
+    justify-content: ;
+    space-between 	Distribuye los ítems dejando (el mismo) espacio entre ellos.
+    space-around 	Distribuye los ítems dejando (el mismo) espacio a ambos lados de cada uno de ellos.
+*/
+
+header{
+    /*border: solid 5px black;*/
+    display: flex;
+    padding: 10px;
+    margin: 25px;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.tittle{
+    width: 30%;
+}
+
+.description{
+    width: 50%;
+}
+
+
+/* ----------------- Overlay hover effect ----------------- */
+
+.image{
```

Comment:
fijate que en el caso de flexbox, si la pantalla es lo suficientemente ancha no quedan 3 columnas como pide el ejercicio. Se te ocurre una forma de solucionarlo?

Reply:
Recien probe hacer tres section (dentro del .container) con la clase column, en cada section puse dos imagenes y le di este estilo:
.column {
    padding: 15px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    max-width: 30%;
}  

Tambien le agregue unos media query y poco mas, lo volvi a subir cualquier cosa.

https://github.com/MatiasGalvan/startup/tree/master/00-HTML-CSS-basics/The%20basics%20of%20CSS/2.5.2%20Flexbox